site/pages/api/cookies.ts

```diff
@@ -0,0 +1,35 @@
+import { NextApiResponse, NextApiRequest } from "next";
+import { COOKIES_POLICY_COOKIE, COOKIE_PREFERENCES_COOKIE, oneYearInSeconds } from "../../constants";
+import { CookiePolicy } from "../../interfaces";
+import { redirectTo, redirectToError, setCookieOnResponseObject } from "../../utils/apiUtils";
+
+const cookies = (req: NextApiRequest, res: NextApiResponse): void => {
+    try {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
Rather than disable the linting here we could create an interface which extends `NextApiRequest` and adds in the expected items in the body, something like:
```
interface CookiesApiRequest extends NextApiRequest {
    body: {
        tracking: "on" | "off";
    };
}
```

Reply:
done